EAD_SESSION_3/Html Files/rtest.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
Copy paste is not a good thing, you have to update the file. Not just let it be in the state.